docs/modules/ROOT/pages/index.adoc

```diff
@@ -484,7 +592,10 @@ A Network configuration defines connection details and operational parameters fo
   "rpc_urls": [
     {
       "type_": "rpc",
-      "url": "https://soroban.stellar.org",
+      "url": {
```

Comment:
I am thinking about option to minimise breaking change and to reuse some env values. 

WDYT about:

```
[
{
"type_": "rpc",
"url": "",
"weight": 100
},
{
"type_": "rpc",
"url": "https://provider.com/sdgdg{SECRET}",
"weight": 100,
"secret": {
  "type": "environment"
  "value": "PROVIDER_SECRET"
 } 
},

]
```

This way users could use simpler way of defining rpc url if they are okay with low security around them and if they do not include secrets.

If rpc includes secrets we can obtain secret from some source and inject into url at place where defined.

This would not be breaking change.

Reply:
I responsed [here](https://github.com/OpenZeppelin/openzeppelin-monitor/pull/199#discussion_r2073074399)